cmd/build/v2/services.go

```diff
@@ -47,7 +47,6 @@ func (bc *OktetoBuilder) GetServicesToBuild(ctx context.Context, manifest *model
 	close(toBuild)
 
 	if len(toBuild) == 0 {
-		oktetoLog.Information("Images were already built. To rebuild your images run 'okteto build' or 'okteto deploy --build'")
```

Comment:
This one is the only one I think we can miss. This can guide/help the users to find out why the latest changes of a service are not being deployed

Reply:
I agree with nacho about having this message. I don't think we can miss this message without affecting our users